server/app/mutations/grid_secrets/common.rb

```diff
@@ -0,0 +1,12 @@
+module GridSecrets
+  module Common
+    include Workers
+
+    def refresh_grid_services(secret_name)
+      GridService.where(:'secrets.secret' => secret_name).each do |grid_service|
```

Comment:
This must be scoped to specific grid, we don't want to update accidentally services in all the grids.